### PR TITLE
Translation of CVE-2021-41817 to spanish

### PR DESCRIPTION
**Pull request by vtamara**

### CODE REVIEW

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+layout: news_post
+title: "CVE-2021-41817: Vulnerabilidad de denegación de servicio por Expresiones Reguales en los métodos para reconocer fechas"
+author: "mame"
+translator: vtamara
+date: 2021-11-15 08:00:00 +0000
+tags: security
+lang: es
+---
+
+Hemos publicado la gema `date` con versiones 3.2.1., 3.1.2, 3.0.2 y 2.0.1
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:11]: There's a typographical error, it should be:

```suggestion
Hemos publicado la gema `date` con versiones 3.2.1, 3.1.2, 3.0.2 y 2.0.1
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+que incluyen una corrección de seguridad a una vulnerabilidad de denegación
+de servicio por expresiones regulares (ReDoS) en los métodos para reconocer
+fechas.  Un atacante podría explotar esta vulnerabilidad para generar
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:14]: It should be a blank line:

```suggestion
fechas.
Un atacante podría explotar esta vulnerabilidad para generar
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+una Denegación de Servicio efectiva.
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:15]: I think it should be:

```suggestion
un ataque de denegación de servicio efectivo.
```

What do you think?

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Los métodos para reconocer fechas incluyendo `Date.parse` usan
+expresiones regulares internamente, algunas de las cuales son vulnerables
+a denegación de servicio por expresiones regulares.
+Las aplicaciones y librerías que emplean tales métodos con entradas
+no confiables pueden verse afectadas.
+
+La corrección limita el tamaño de la entrada a 128 bytes de manera
+predeterminada en lugar de cambiar las expresiones regulares.
+Esto es porque la gema Date usa muchas Regexps y es posible que aún haya
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:30]: I think it should be:

```suggestion
Esto es porque la gema Date usa muchas expresiones regulares y es posible que aún haya
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+vulnerabilidades no descubiertas en algunas.  Por compatibilidad, se
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:31]: There's an additional space, it should be:

```suggestion
vulnerabilidades no descubiertas en algunas. Por compatibilidad, se
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+permite eliminar la limitación pasando explícitamente la palabra
+reservada `limit` en `nil`, como en `Date.parse(str, limit: nil)`,
+pero tenga en cuenta que el reconocimiento puede tardar un largo tiempo.
+
+Por favor actualice la gema date a una de las versiones 3.2.1, 3.1.2, 3.0.2,
+2.0.1 o posterior.  Puede usar `gem update date` para actualizarla.
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:37]: There's an additional space, it should be:

```suggestion
2.0.1 o posterior. Puede usar `gem update date` para actualizarla.
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+De forma alternativa, puede actualizar Ruby a 3.0.3, 2.7.5, 2.6.9 o posterior.
+
+## Versiones afectadas
+
+* gema date 2.0.0 y anteriores (distribuidas con la serie Ruby 2.6 antes de Ruby 2.6.9)
+* gema date 3.0.1 y anteriores (distribuidas con la serie  Ruby 2.7 antes de Ruby 2.7.5)
+* gema date 3.1.1 y anteriores (distribuida con la serie Ruby 3.0 antes de Ruby 3.0.3)
+* gema date gem 3.2.0 y anteriores
+
+## Creditos
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:48]: There's a typographical error, it should be:

```suggestion
## Créditos
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+Agradecimos a [svalkanov](https://github.com/SValkanov/) por descubrir
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md:50]: I think it should be:

```suggestion
Agradecemos a [svalkanov](https://github.com/SValkanov/) por descubrir
```

File: es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+el problema.
+
+## Historia
+
+* Publicado originalmente en inglés el 2021-11-15 08:00:00 (UTC)
+* Mención sobre nuevas versiones de Ruby el 2021-11-24 13:20:00 (UTC)
+
```
Comment (gfvcastro) [inline es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md]: There's an additional blank line, it should removed.

Comment (vtamara) [reply es/news/_posts/2021-11-15-date-parsing-method-regexp-dos-cve-2021-41817.md]: I had already removed it. Thanks for suggestions. I applied all of them.

**Pull request merged**